bcipy/helpers/convert.py

```diff
@@ -210,3 +214,79 @@ def edf_annotations(triggers: List[Tuple[str, str, float]],
     """
     return [(timestamp, durations.get(targetness, 0.0), label)
             for (label, targetness, timestamp) in triggers]
+
+
+def compress(tar_file_name: str, members: List[str]) -> None:
+    """Compress.
+       Adds files (members) to a tar_file and compresses them using gzip
+
+    tar_file_name (str): the name of resulting compressed file
+    """
+
+    for member in members:
+        # Checks if file exists
+        if not os.path.exists(member):
+            raise FileNotFoundError(f"This file or folder, '{member}', "
+                                    "does not exist!\nPlease rerun program")
+        # Checks for file names that may be too long. OS level restriction.
+        if len(member) > FILE_LENGTH_LIMIT:
+            logger.warning(
+                f'File length exceeds compression limit=[{FILE_LENGTH_LIMIT}]. '
+                'This may cause issues later with extraction. Please proceed at your own discretion.')
+
+    full_tar_name = f'{tar_file_name}.tar.gz'
+    if os.path.exists(full_tar_name):
+        raise Exception(f"This tar archive=[{full_tar_name}] already exists, continuing will "
+                        "overwrite anything in the existing archive.")
+
+    # Opens file for gzip (gz) compressed writing (w)
+    # Uses default compression level 9 (highest compression, slowest speed)
+    with tarfile.open(full_tar_name, mode="w:gz") as tar:
+        # Sets progress bar through tqdm
+        progress = tqdm(members)
+        for member in progress:
+            # Adds file/folder to the tar file and compresses it
+            tar.add(member)
+            # Sets progress description of progress bar
+            progress.set_description(f"Compressing {member}")
+
+
+def decompress(tar_file: str, path: str) -> None:
+    """Extracts tar_file and places specified "members" in "path"
+       If members remains None, all members will be extracted
+    """
+
+    full_tar_name = f'{tar_file}.tar.gz'
```

Comment:
It's not clear from the method documentation that you only want the base name of the file. You should either update the documentation or make the implementation more lenient by checking for the suffix (or stripping it) before adding it here. The same comment applies to the other methods. Given the repetition, that code could be factored out into another function.